core-bundle/contao/dca/tl_form_field.php

```diff
@@ -380,6 +381,25 @@
 			'eval'                    => array('fieldType'=>'radio', 'filesOnly'=>true, 'mandatory'=>true, 'tl_class'=>'clr'),
 			'sql'                     => "binary(16) NULL"
 		),
+		'altchaAuto' => array
+		(
+			'inputType'               => 'select',
+			'options'                 => array('onfocus', 'onload', 'onsubmit'),
```

Comment:
```suggestion
			'options'                 => array('onfocus', 'onsubmit'),
```
I’d fear that admins would select `onload` here without realizing what harm this would cause.

If someone knows what they are doing, they can still adjust the DCA to allow `onload`.

Reply:
Changed in f7d0dff93eddaf5c4a925a54d95784347d4cbdd1.